articles/active-directory-b2c/date-transformations.md

```diff
@@ -206,3 +206,40 @@ To run the claim transformation, you first need to get the current dateTime and
     - **timeSpanInSeconds**: 7776000 (90 days)
 - Output claims:
     - **result**: true
+
+## IsTermsOfUseConsentRequired
+
+Determine whether the terms of use updated date is newer than the terms of use consented date. The result is a new boolean ClaimType with a value of `true` or `false`.
```

Comment:
Dear @bolt-io, thanks for your contribution. I'd prefer to make this CT more generic. Please consider updating this paragraph with the following one:

Determine whether a date claim type is earlier, or greater than a specific date. The result is a new boolean claim type with a value of `true` or `false`.

Reply:
updated 👍 Thanks